src/backend/cuda/platform.cpp

```diff
@@ -146,6 +146,18 @@ static inline string toString(T val)
     return s.str();
 }
 
+static inline
+int getMinSupportedCompute(int cudaMajorVer)
+{
+    // Vector of minimum supported compute versions
+    // for CUDA toolkit (i+1).* where i is the index
+    // of the vector
+    static const vector<int> minSV{1, 1, 1, 1, 1, 1, 2, 2, 3, 3};
```

Comment:
A std::array object would be better for this. No need to allocate on the heap if we are not sending references to the values.